tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -4321,6 +4522,25 @@ wan/lacp/test_wan_lag_min_link.py::test_lag_min_link:
       - "len(minigraph_portchannels) < 2"
       - "not any(len(v['members']) > 1 for _, v in minigraph_portchannels.items())"
 
+#######################################
+#####           wan_test         #####
+#######################################
+wan_test:
+  skip:
+    reason: "It is not tested for now"
+    conditions:
+      - True
+
+#######################################
+#####           wol          #####
+#######################################
+
+wol:
```

Comment:
please remove wol, which is only for mx and m0, TH5 should not run this script at all.